app/javascript/reducers/answers.js

```diff
@@ -108,6 +110,30 @@ function answerFetchFailure(state, action) {
   };
 }
 
+/**
+ * Adds answers from batch creation
+ */
+export function addAnswers(state, action) {
+  return {
+    ...state,
+    [action.buildingId]: {
```

Comment:
This isn't adding answers into Redux correctly - the `state` that's received by the answers reducer is actually just the `answers` object within that specific building, not `buildings` itself. So this should be:
```
return {
  ...state,
  ...action.answers
}